docs/build/isc/v1.1/docs/how-tos/core-contracts/nft/mint-nft.md

```diff
@@ -19,12 +19,20 @@ The Stardust update allows you to create your own NFTs. You can also use [IRC27]
 
 2. Get the senders AgentID:
 
+The `ISCAgentID` represents the identifier of the agent (user or contract) whose NFTs you want to retrieve. You can get the `AgentID` from the sender by calling `ISC.sandbox.getSenderAccount()`.
+
 ```solidity
 ISCAgentID memory agentID = ISC.sandbox.getSenderAccount();
 ```
 
 3. Create an `IRC27Metadata` struct  with all the needed data:
 
+:::tip 
+You can refer to [Get NFT Metadata guide](https://wiki.iota.org/isc/how-tos/core-contracts/nft/get-nft-metadata/) to understand how to create an `IRC27Metadata` 
```

Comment:
The update looks great to me. However, I recall that when I linked to a guide on the wiki using its HTTPS URL, Jeroen changed it to the following format:

`../../../reference/#`

Perhaps you should link to the get-nft-metadata guide internally from the wiki repo instead of using the HTTPS URL.

@lucas-tortora, could you please confirm if linking to the get-nft-metadata guide internally, as Vivek has implemented, is the best approach?

Well done, Vivek!